### PR TITLE
Clarify unsafe recovery cmd input requirements

### PR DESCRIPTION
**Pull request by v01dstar**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Clarify the requirement of unsafe recovery.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.4 (TiDB 6.4 versions)
- [x] v6.3 (TiDB 6.3 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by Connor1996 — COMMENTED** [review]
丰富下第3步，怎么用 table id 查 table name，以及具体的 ADMIN CHECK/RECOVER/ClEANUP 命令

**Comment by v01dstar** [comment]
> 丰富下第3步，怎么用 table id 查 table name，以及具体的 ADMIN CHECK/RECOVER/ClEANUP 命令

Done. Additionally, I replaced `ADMIN RECOVER` and `ADMIN CLEANUP` with index recreation. Because, in reality, index recreation is more comprehensive and faster (multi-threaded).

File: online-unsafe-recovery.md [hunk 1]
```diff
@@ -161,12 +170,44 @@ PD 下发恢复计划后，会等待 TiKV 上报执行的结果。如上述输
...
 
+执行完成后，数据和索引可能会不一致。请使用 SQL 的 [`ADMIN CHECK`](/sql-statements/sql-statement-admin-check-table-index.md) 对受影响的表进行数据索引一致性检查。
+
+{{< copyable "sql" >}}
+
+```sql
+ADMIN CHECK TABLE table_name;
+```
+
+之后可以通过重命名旧索引，创建新索引，删除旧索引的方式来修复数据索引不一致的问题。
```
Comment (Connor1996) [inline online-unsafe-recovery.md:185]: 用 ADMIN RECOVER 和 CLEANUP

Comment (v01dstar) [reply online-unsafe-recovery.md:185]: 真实环境测试反应，`ADMIN RECOVER` 和 `ADMIN CLEANUP` 因为不是并行的，所以相对于重建要慢很多，因此目前建议创建而不是修复。

Comment (Connor1996) [inline online-unsafe-recovery.md:185]:
```suggestion
若结果有不一致的索引，可以通过重命名旧索引，创建新索引，删除旧索引的方式来修复数据索引不一致的问题。
```

File: online-unsafe-recovery.md [hunk 2]
```diff
@@ -145,6 +145,15 @@ PD 下发恢复计划后，会等待 TiKV 上报执行的结果。如上述输
...
 
+得到受影响的 tables' id 后，可以使用 information schema 来查看受影响的表名。
+
+{{< copyable "sql" >}}
+
+```sql
+
+SELECT TABLE_SCHEMA, TABLE_NAME, TIDB_TABLE_ID FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_ID IN (64, 27);
+```
+
```
Comment (Oreoxmt) [inline online-unsafe-recovery.md:156]:
````suggestion
得到受影响的 table id 后，可以使用 `INFORMATION_SCHEMA` 来查看受影响的表名。

```sql
SELECT TABLE_SCHEMA, TABLE_NAME, TIDB_TABLE_ID FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_ID IN (64, 27);
```

````

File: online-unsafe-recovery.md [hunk 3]
```diff
@@ -161,12 +170,44 @@ PD 下发恢复计划后，会等待 TiKV 上报执行的结果。如上述输
...
 >
 > 数据可以读写并不代表没有数据丢失。
 
+执行完成后，数据和索引可能会不一致。请使用 SQL 的 [`ADMIN CHECK`](/sql-statements/sql-statement-admin-check-table-index.md) 对受影响的表进行数据索引一致性检查。
+
+{{< copyable "sql" >}}
+
+```sql
+ADMIN CHECK TABLE table_name;
+```
```
Comment (Oreoxmt) [inline online-unsafe-recovery.md:183]:
````suggestion
执行完成后，数据和索引可能会不一致。请使用 [`ADMIN CHECK`](/sql-statements/sql-statement-admin-check-table-index.md) 对受影响的表进行数据索引的一致性检查。

```sql
ADMIN CHECK TABLE table_name;
```
````

File: online-unsafe-recovery.md [hunk 4]
```diff
@@ -161,12 +170,44 @@ PD 下发恢复计划后，会等待 TiKV 上报执行的结果。如上述输
...
+
+删除旧索引：
+
+{{< copyable "sql" >}}
+
+```sql
+ALTER TABLE table_name DROP INDEX index_name_lame_duck;
+```
+
```
Comment (Oreoxmt) [inline online-unsafe-recovery.md:210]:
````suggestion
若结果有不一致的索引，可以通过重命名旧索引、创建新索引、删除旧索引的方式来修复数据索引不一致的问题。

重命名旧索引：

```sql
ALTER TABLE table_name RENAME INDEX index_name TO index_name_lame_duck;
```

创建新索引：

```sql
ALTER TABLE table_name ADD INDEX index_name (column_name);
```

删除旧索引：

```sql
ALTER TABLE table_name DROP INDEX index_name_lame_duck;
```

````

**Review by Oreoxmt — COMMENTED** [review]
Rest LGTM

**Pull request merged**